src/twisted/names/test/test_names.py

```diff
@@ -1211,7 +1213,7 @@ class BindAuthorityTests(unittest.TestCase):
     Tests for L{twisted.names.authority.BindAuthority}.
     """
 
-    def loadBindString(self, s):
+    def loadBindString(self, s: bytes, path: AnyStr) -> authority.BindAuthority:
```

Comment:
```suggestion
    def loadBindString(self, s: bytes, path: AnyStr = '') -> authority.BindAuthority:
```

Reply:
Good suggestions, will implement them.

Reply:
Hey @adiroiban setting path to '' or b'' by default is not working. Mypy is complaining and tests are failing. 

```
src/twisted/names/test/test_names.py:1216:55: error: Incompatible default for argument "path" (default has type "str", argument has type "bytes")  [assignment]
src/twisted/names/test/test_names.py:1230:20: error: Incompatible types in assignment (expression has type "bytes", variable has type "str")  [assignment]
```

So I solved this issue a different way. I would be glad to hear your suggestion for changes. I just converted path to be pathType, by default its str but you can make it bytes. Its not the most elegant solution though.